content/docs/tech-insights/track-datadog-slos/index.md

```diff
@@ -0,0 +1,123 @@
+
+### **Prerequisites**
+
+Before you begin, make sure you have the following:
+
+- A Datadog account: Sign up for a Datadog account if you don't have one already.
+- Access to your system components in Datadog: Ensure you have the necessary permissions to access the components you want to track SLOs from.
+- Defined SLOs in at least one of the components in Datadog.
+- Annotations configured in your `catalog-info.yaml` file in your repository. These are the annotations used in our Datadog built-in Data Source to track SLOs and Monitors:
+  - `datadoghq.com/slo_tag`: The Datadog SLO tag to use for this component. This is the tag that will be used to filter the components that have SLOs defined.
+  - `datadoghq.com/monitor_tags`: The Datadog SLO target to use for this component. This are the monitor tags that will be used to filter the components that have Monitors defined in Datadog.
+
+### **What you'll learn**
+
+With Roadie's Tech Insights feature, there's an easier way. In this tutorial, we’re going to:
+
+1. Set up Datadog integration.
+2. Automatically scan Components in the Backstage catalog and record how many SLOs are defined for each.
+3. Visualize the distribution of SLOs defined count.
+4. Create a check to verify if at least one SLO is defined for each component.
+
+As we go through this process, you’ll learn:
+
+1. How to use built-in Data Sources (Datadog) and how to create Checks with Tech Insights.
+2. How to detect which component has SLO defined with Tech Insights.
+3. How to use a comparison operator.
+
+Let's get started.
+
+## Setup Datadog integration
+
+In order to collect the Datadog SLO count from each component, the Datadog integration needs to be set up on Roadie. To do that, follow these steps:
+
+### Administration → Settings → Datadog
+
+Set the hostname of your Datadog app.
+
+### Administration → Settings → Secrets
+
+Set the `Datadog API Token` and `Datadog APP Token` in order for your Roadie instance to contact your Datadog app.
+
+| Key          | Token             |
+| ------------ | ----------------- |
+| DD_API_TOKEN | Datadog API Token |
+| DD_APP_TOKEN | Datadog APP Token |
+
+## Record the SLO count for each component
+
+In order to know which software has Datadog SLOs configured, we’re going to use the built-in Datadog Data Source in Roadie Tech Insights. It will run periodically and extract the SLO count from each component.
+
+These are the steps to set that up.
+
+1. Visit Tech Insights and click into the Data Sources tab. Search for the Datadog Data Source from the Data Sources list and select it.
+
+   ![Data Source Listing](./datasources_list.png)
+
+2. You may need to wait some time for the data source to collect all SLOs and Monitors from Datadog. It must contact the Datadog APIs for each component, which is captured by the already set filter.
```

Comment:
Might be useful to add more details on what this filter is about (who sets it, when, can it be changed).